### PR TITLE
[increment patch] Added reorder first and last and only item logic

### PR DESCRIPTION
**Pull request by zinabat**
## Context
[DE39891](https://rally1.rallydev.com/#/detail/defect/406934850716?fdp=true)
[DE40001](https://rally1.rallydev.com/#/detail/defect/409224266100?fdp=true)

* Adds logic to show only relevant "Move Up" and "Move Down" actions
* Hides context menu if there are no secondary actions and there is only one item in the list
* Moves the `secondary-action` slot to below the reorder actions

### First Item
Screen Shot 2020-08-10 at 1 48 25 PM

### Last Item
Screen Shot 2020-08-10 at 1 48 32 PM

### Only Item
Screen Shot 2020-08-10 at 1 49 00 PM

### Menu item order
Screen Shot 2020-08-10 at 2 09 07 PM

## Quality
* New unit tests added for these scenarios
* Tested with demo page

### CODE REVIEW

File: test/list-item-accumulator.test.js [hunk 1]
```diff
@@ -17,4 +17,41 @@ describe('d2l-labs-list-item-accumulator', () => {
...
+			<d2l-labs-list-item-accumulator draggable="true" key="1"></d2l-labs-list-item-accumulator>
+			<d2l-labs-list-item-accumulator draggable="true" key="2"></d2l-labs-list-item-accumulator>
+			<d2l-labs-list-item-accumulator draggable="true" key="3"></d2l-labs-list-item-accumulator>
+		</div>`;
+
+		it('should only show "Move Down" when first item', async() => {
+			const el = await fixture(basicList);
+			const items = el.firstElementChild.shadowRoot.querySelectorAll('d2l-menu-item');
+			expect(items).to.not.be.null;
+			expect(items).to.have.lengthOf(1);
```
Comment (AKobets) [inline test/list-item-accumulator.test.js:32]: is it possble that the `d2l-labs-list-item-accumulator` will have menu options that are not related to move up/down?
Should test assertions be more flexible ?

Comment (zinabat) [reply test/list-item-accumulator.test.js:32]: I could maybe check that the `d2l-menu-item` nodes only contain the action we expect... I'll fiddle with it

Comment (zinabat) [reply test/list-item-accumulator.test.js]: I've altered the tests so that they check for explicit existence or non-existence of the "Move Up" and "Move Down" items which should allow more flexibility
